src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -866,6 +868,8 @@ public static void copyFile(final File srcFile, final File destFile, final CopyO
      * @since 2.1
      */
     public static long copyFile(final File input, final OutputStream output) throws IOException {
+        Objects.requireNonNull(input, "input");
+        Objects.requireNonNull(output, "output");
```

Comment:
-1: no need to check on `output`, this already done in `copyLarge`.

I'll stop this type of comment since these checks over the top in this PR. IOW, let's not check for NPEs when we do not need to.

Reply:
@garydgregory The other cases before I understood the pattern behind "-1". In this case a FileInputStream is created, afterwards copyLarge calls copy calls copyLarge which then does the check.
I'll fix the former cases and wait with removal for cases like this on your comment.